### DISCUSSION TITLE
网页“更多特性”专区优化并完成相关开发

### DISCUSSION THREAD

**Discussion by WyElaine** [halo-dev/halo#5567] [General]
### 你当前使用的版本

2.13.1

### 描述一下此特性

<img width="1440" alt="截屏2024-03-21 20 21 47" src="https://github.com/halo-dev/halo/assets/164335742/21b63657-cfc6-40d8-9642-b5f786f5da87">
首页的“更多特性”专区不可以操作 希望可以深入 或者可以完成跳转相关功能的展示

**Comment by ruibaby** [comment]
这是 Halo 自用的官网，这和 Halo 仓库本身没有任何关系，请不要提交和仓库本身无关的 issue。

/triage unresolved
